deploy/infrastructure/terraform/.env

```diff
@@ -0,0 +1,35 @@
+# Can be set to your own hook builds
```

Comment:
im not too excited about duplicate `.env` files to maintain. any chance we could have only one?

Reply:
Yeah I'm not happy about it either, I can look into options for merging them into one but I don't know of an easy solution based on .env files. 

There's a few bits of reused variables and bash code in the project, I was thinking of converting the bash provisioning to Ansible and splitting out the more specific provisioning tasks with roles based on whether the machine is bare metal, equinix, vagrant etc.. Though I think that would be a separate issue/PR for later on.

Reply:
thanks for the response and especially for your work and patience on this!